channel-connector-starter/pom.xml

```diff
@@ -33,7 +32,7 @@
     <dependency>
       <groupId>com.salesforce.einsteinbot</groupId>
       <artifactId>einstein-bot-sdk-java</artifactId>
-      <version>1.0.0-SNAPSHOT</version>
+      <version>1.0.0</version>
```

Comment:
Shouldn't we use `1.0.1` version of `einstein-bot-sdk-java`? If that's not published yet to maven, then we should wait and then merge this. Or are you planning to cut a new version of connector to bump sdk in connector?

Reply:
`1.0.0` is the current release version of SDK. `1.0.0` connector is also published which depends on `1.0.0` of SDK. Why do you think we should use `1.0.1`?

Reply:
That's why:
https://github.com/forcedotcom/einstein-bot-sdk-java/blob/fea93b66994afb85dd660c92d396a96f741d997a/pom.xml#L14